src/main/java/com/example/gildedrose/GildedRoseService.java

```diff
@@ -0,0 +1,26 @@
+package com.example.gildedrose;
+
+import com.example.gildedrose.item.*;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+@Service
+public class GildedRoseService {
+    private GildedRose gildedRose;
+
+    public GildedRoseService() {
+        this.gildedRose = new GildedRose(new ArrayList<>(Arrays.asList(
```

Comment:
**question**: Do we need the `GildedRose` for this service?

We are only using the new `getItems`. It feels like we might be violating one or more a SOLID princples.

Reply:
I guess we could just pass the items to GildedRoseService?